src/main/java/com/powsybl/openloadflow/network/impl/LfNetworkLoaderImpl.java

```diff
@@ -621,6 +621,12 @@ private static void createTransformerVoltageControl(LfNetwork lfNetwork, RatioTa
         double targetValue = rtc.getTargetV() / regulatingTerminalNominalV;
         Double targetDeadband = rtc.getTargetDeadband() > 0 ? rtc.getTargetDeadband() / regulatingTerminalNominalV : null;
 
+        if (!VoltageControl.checkTargetV(targetValue, controlledBus.getNominalV(), parameters)) {
+            LOGGER.warn("RatioTapChanger on transformer '{}' has an inconsistent target voltage: {} pu: incremental voltage control discarded", controllerBranchId, targetValue);
+            rtc.setRegulating(false);
```

Comment:
no, don't modify the iIDM network. the regulator is disabled only internally in OLF.